app/models/Status.scala

```diff
@@ -17,38 +17,107 @@
 package models
 
 import play.api.libs.json._
+import uk.gov.hmrc.wco.dec.Response
 
 sealed trait Status
 
 object Status {
 
   implicit object StatusFormat extends Format[Status] {
     def reads(status: JsValue): JsResult[Status] = status match {
-      case JsString("Accepted")  => JsSuccess(Accepted)
-      case JsString("Arrived")   => JsSuccess(Arrived)
-      case JsString("Departed")  => JsSuccess(Departed)
-      case JsString("PreLodged") => JsSuccess(PreLodged)
-      case JsString("Query")     => JsSuccess(Query)
-      case _                     => JsError("Incorrect value")
+      case JsString("01")   => JsSuccess(Accepted)
```

Comment:
why Jsstring?

Reply:
so we're sending this as Json, the values will be JsString, code inside will be the same like in notifications